geth/account/accounts.go

```diff
@@ -30,14 +25,20 @@ var (
 
 // Manager represents account manager interface
 type Manager struct {
-	nodeManager     common.NodeManager
+	node            accountNode
 	selectedAccount *common.SelectedExtKey // account that was processed during the last call to SelectAccount()
+
+	extKeyImporter   extendedKeyImporter
+	subAccountFinder subAccountFinder
 }
 
 // NewManager returns new node account manager
 func NewManager(nodeManager common.NodeManager) *Manager {
+	node := newAccountNodeManager(nodeManager)
 	return &Manager{
-		nodeManager: nodeManager,
+		node:             node,
+		extKeyImporter:   new(extendedKeyImporterBase),
```

Comment:
Why this `...Base`? Will there be something larger founded on this `...Base`? Also first `extKeyImporter` and then `extendedKeyImporterBase`. Not consistent.

Reply:
Ok, i think i should change it.